ui/pages/snaps/snap-view/index.scss

```diff
@@ -0,0 +1,111 @@
+.snap-view {
+  $height-screen-sm-max: 100%;
```

Comment:
Do we need all of this extra CSS? If so, why? And would it be possible to use design system components directly instead? 🤔

Reply:
In order to make <Page> component to follow the responsive behaviour of the header, I needed to use this extra CSS I took from the header (with account and network). 

I couldn't find another way. If this extra CSS is not added, the Page component will have it's own width that mismatch the header width and cause everything to look like a broken UI (mess).